manage.py

```diff
@@ -17,6 +17,10 @@
 CONFIG_FILE.read('app.cfg')
 
 CLIENTS_DATA = yaml.load(open('clients.yaml')) or {}
+# remove deprecated configuration key
+for data in CLIENTS_DATA:
+    if 'redirect_uri' in data:
```

Comment:
At this point, it's checking if `redirect_uri` is in `data` (which is a `str`), needs to be:
```
for data in CLIENTS_DATA:
    if 'redirect_uri' in CLIENTS_DATA[data]:
        del CLIENTS_DATA[data]['redirect_uri']

```